js/main.js

```diff
@@ -71,7 +72,26 @@ async function main() {
   getCSVData(async (cards_data) => {
     switch (CURRENT_PAGE) {
       case "/gacha.html":
-        GACHA_BUTTON.onclick = (event) =>
+        GACHA_DISPLAY.forEach((elem) => {
+          elem.addEventListener("change", function(e) {
+            gacha_display_selection = e.target.value;
+            if(gacha_display_selection == "gacha-grid"){
+              GACHA_SECTION.classList.add("grid-display");
+              GACHA_SECTION.classList.remove("pile-display");
+              addHover();
+            } else {
+              GACHA_SECTION.classList.remove("grid-display");
+              GACHA_SECTION.classList.add("pile-display");
+              removeHover();
+            }
+          });
+        });
+        if(window.innerWidth <= 800){
+          document.getElementById("gacha-grid").click();
+        } else {
+          document.getElementById("gacha-pile").click();
+        }
```

Comment:
I don't think we need this, unless Bing specifically requested it.

If you want to keep it, is it possible to set the value of the input directly instead of doing it through a "click"? This can be addressed later, just let me know and we'll add it to the Notion

Reply:
This makes sense to me to be different on mobile devices since I don't feel like the pile layout is ideal with that. I have updated the way this is done though to not use a click, and instead to check it with more typical JavaScript and then just dispatch a new change event.